package/MDAnalysis/analysis/align.py

```diff
@@ -424,16 +431,18 @@ def alignto(mobile, reference, select="all", mass_weighted=False,
                                                  tol_mass=tol_mass,
                                                  strict=strict)
 
-    if mass_weighted:
-        # division by the mean is done in rmsd, not done in qcp, but is done in
-        # _fit_to
+    if mass_weighted is not None:
+        warnings.warn("mass weighted is deprecated argument. Please use "
+                      " 'weights=\"mass\" instead. Will be removed in 0.17.0",
+                      category=DeprecationWarning)
+        if mass_weighted:
+            weights = 'mass'
+
+    if weights == 'mass':
```

Comment:
Btw this method leaves a nasty warning right now when weights is a numpy array since the equal operator tries a elementwise operation, this is the only comparison that will happen in the future. Anyone a idea how to nicely express the logic that tests for an array_like?